swagger.yaml

```diff
@@ -230,3 +230,24 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Happening'
+  /happenings/{id}:
+    delete:
+      tags:
+        - Happening
+      summary: Removes happening.
+      security:
+        - JWT: []
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                id:
+                  type: string
+                  format: uuid
+                  example: 728f10a1-94a5-4049-b25e-4eadce9be938
+      responses:
+        200:
+          description: Happening has been removed.
```

Comment:
standard response JSON structure is missing and the status code doesn't correspond to the one being set in web controller